### PR TITLE
Added conversion for C++17 language standard

### LINKED ISSUE #518: CXX_LANGUAGE_STANDARD is not migrated by `Jucer2Reprojucer`
I converted an existing project with `Jucer2Reprojucer`. The CXX_LANGUAGE_STANDARD is not migrated and appears as an empty, commented-out setting in the generated CMakeLists.txt.

### PR DESCRIPTION
**Pull request by TheSlowGrowth**
This should fix #518.

### CODE REVIEW

File: Jucer2Reprojucer/main.cpp [hunk 1]
```diff
@@ -704,6 +704,9 @@ int main(int argc, char* argv[])
                        if (value == "14")
                          return "C++14";
 
+                       if (value == "17")
+                         return "C++17";
```
Comment (McMartin) [inline Jucer2Reprojucer/main.cpp]: Nice! :+1:

File: Jucer2Reprojucer/main.cpp [hunk 2]
```diff
@@ -1619,6 +1622,8 @@ int main(int argc, char* argv[])
                                       return "(default)";
                                     if (value == "stdcpp14")
                                       return "C++14";
+                                    if (value == "stdcpp17")
+                                      return "C++17";
```
Comment (McMartin) [inline Jucer2Reprojucer/main.cpp]: The Visual Studio exporter setting "C++ standard to use" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-2cd603f4b77a06ba4f2d589cda677923L1863-L1865. At that time, Projucer didn't know about C++17 at all, so these 2 lines are actually dealing with unexpected input. Please remove them.

File: Jucer2Reprojucer/main.cpp [hunk 3]
```diff
@@ -1641,6 +1646,9 @@ int main(int argc, char* argv[])
                                   if (value == "-std=c++14")
                                     return "C++14";
 
+                                  if (value == "-std=c++17")
+                                    return "C++17";
+
```
Comment (McMartin) [inline Jucer2Reprojucer/main.cpp]: The Linux Makefile exporter setting "C++ standard to use" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-c0f0b20e4d0da4c5c89c5944c3506975L361-L368. At that time, Projucer didn't know about C++17 at all, so these 3 lines are actually dealing with unexpected input. Please remove them.

File: Jucer2Reprojucer/main.cpp [hunk 4]
```diff
@@ -1940,6 +1948,12 @@ int main(int argc, char* argv[])
                                     if (value == "gnu++14")
                                       return "GNU++14";
 
+                                    if (value == "c++17")
+                                      return "C++17";
+
+                                    if (value == "gnu++17")
+                                      return "GNU++17";
+
```
Comment (McMartin) [inline Jucer2Reprojucer/main.cpp]: The Xcode exporter configuration setting "C++ Language Standard" was replaced by the project setting "C++ Language Standard" in https://github.com/WeAreROLI/JUCE/commit/2e81e8a0dc8e4c509baa3317ad760153ddb2eeb5#diff-29de8ceb6da4050657ca1e601ef42774L458-L468. At that time, Projucer didn't know about C++17 at all, so these 6 lines are actually dealing with unexpected input. Please remove them.

**Comment by TheSlowGrowth** [comment]
Okay, I didn't know that. Thanks for the clarification. I feel sorry for the additional overhead - it probably would've been way quicker for you to just make the change yourself... But it helps me to understand FRUT a little better, so hopefully it's worth it ;-)

I squashed my changes into the original commit and force-pushed. Should be fine now, I hope.

**Comment by McMartin** [comment]
I'm really glad when people get involved and contribute to FRUT. It doesn't matter how much explaining I have to do, it's always worth it!

The diff looks very good now. Thanks for squashing and force-pushing!

I have a couple of comments on the commit message: please make the commit title start with an infinitive verb (`Add` instead of `Added`), and write the commit message body with complete English sentences (which start with a capital letter and end with a period). That way it will match the commit messages of all commits in FRUT.

As a side note (feel free to ignore), you might also want to commit using the same email address that you are using with the @TheSlowGrowth account instead of @dummygithubaccount, since I will add @TheSlowGrowth to the [contributors table](https://github.com/McMartin/FRUT#contributors) to credit you for bug reporting (🐛) and code (💻).

**Pull request merged**